cli/unstable_spinner.ts

```diff
@@ -139,8 +139,7 @@ export class Spinner {
 
   #interval: number;
   #color: Color | undefined;
-  #intervalId: number | undefined;
-  #active = false;
+  #intervalId: number | null = null;
```

Comment:
nit: why do you change this type?

Reply:
Because `this.#intervalId` is unset  in `stop()` . I think `null` shows more clearly that it is intentionally unset than `undefined`. Also because of the nature how js works, `this.#intervalId` cannot be checked with `if (!this.#intervalId)` anymore since the value `0` would pass (even though  I think `setInterval` would probably never return 0). So `(this.#intervalId == null)` seemed nicer.

Reply:
If my understanding of what the spec says on setTimeout and setInterval, the ID returned from them has to be greater than zero.

Reply:
> the ID returned from them has to be greater than zero.

That sounds correct

https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#timer-initialisation-steps

> If previousId was given, let id be previousId; otherwise, let id be an [implementation-defined](https://infra.spec.whatwg.org/#implementation-defined) integer that is greater than zero and does not already [exist](https://infra.spec.whatwg.org/#map-exists) in global's [map of setTimeout and setInterval IDs](https://html.spec.whatwg.org/multipage/timers-and-user-prompts.html#map-of-settimeout-and-setinterval-ids).